app/views/service_providers/_service_provider_table.erb

```diff
@@ -3,7 +3,7 @@
 
   <% unless sps[:apps].blank? %>
     <table class="usa-table">
-      <%= render partial: 'service_provider_table_header', locals: { show_created_at: show_created_at } %>
+      <%= render partial: 'service_provider_table_header', locals: { is_prod: sps[:apps][0].prod_config, show_created_at: show_created_at } %>
```

Comment:
I dislike hardcoding array indexes, we can use `.first` here?
```suggestion
      <%= render partial: 'service_provider_table_header', locals: { is_prod: sps[:apps].first.prod_config, show_created_at: show_created_at } %>
```

Reply:
Thanks, I'm still getting used to Ruby, and JS doesn't have that method on arrays. Will update.

Reply:
i think rather than using the `sp[:apps]` data  to determine this, it would be preferable to pass the local in, like we're doing with `show_created_at`. IE:

[index view](https://github.com/18F/identity-dashboard/blob/main/app/views/service_providers/index.html.erb#L3)
`<%= render partial: 'service_provider_table', locals: { service_providers: @service_providers, show_created_at: false, is_prod: false } %>`

i am also noticing that the value of `is_prod` and the value of `show_created_at` will be the same, so maybe we should just rename `show_created_at` to `is_prod` and use that to determine if the `created_at` column and the `accessible in the integration environment` column, rather than having two variables to keep track of.

Reply:
> so maybe we should just rename `show_created_at` to `is_prod`

Yessss!!! This would be a nice update for future us.

Reply:
@Jeremy1026 i think i got confused -- `show_created_at` is admin vs user, not prod vs integration applications. 

@ajfarkas probably the right place to do this would be in the service provider controller, in this array of hashes: https://github.com/18F/identity-dashboard/blob/main/app/controllers/service_providers_controller.rb#L14-L22